src/firmware/linux/host/ioctl.rs

```diff
@@ -10,7 +10,7 @@ use crate::impl_const_id;
 use crate::error::FirmwareError;
 
 #[cfg(feature = "snp")]
-use crate::firmware::host::SnpPlatformStatus;
+use SnpPlatformStatus;
```

Comment:
```suggestion
use super::types::SnpPlatformStatus;
```

I think that would be the path? Try referring to the full path of `SnpPlatformStatus`.

Reply:
Yeah, isn't this imported in line 6. Can probably remove this